scripts/test-examples

```diff
@@ -61,11 +61,16 @@ function bootBroccoli() {
       examplesEnv.EXAMPLES = 1;
 
       var broccoli = childProcess.exec('node ' + broccoliBin + ' serve', { env: examplesEnv });
+      var broccoliOutput = "";
 
       broccoli.stdout.on('data', function (data) {
-        log('broccoli', data);
```

Comment:
`end` only occurs when there is no more data. Broccoli will continuously emit data. From the linked spec:

> The 'end' event is emitted when there is no more data to be consumed from the stream.

This only occurs if the process quits or closes STDOUT, which it doesn't.